cdap-data-fabric/src/main/java/io/cdap/cdap/data2/dataset2/lib/table/leveldb/LevelDBTableService.java

```diff
@@ -109,6 +110,7 @@ public void setConfiguration(CConfiguration config) {
     compressionEnabled = config.getBoolean(Constants.CFG_DATA_LEVELDB_COMPRESSION_ENABLED);
     blockSize = config.getInt(Constants.CFG_DATA_LEVELDB_BLOCKSIZE, Constants.DEFAULT_DATA_LEVELDB_BLOCKSIZE);
     cacheSize = config.getLong(Constants.CFG_DATA_LEVELDB_CACHESIZE, Constants.DEFAULT_DATA_LEVELDB_CACHESIZE);
+    maxOpenFiles = config.getInt(Constants.CFG_DATA_LEVELDB_MAX_OPEN_FILES);
```

Comment:
If it has to be more than 10, could we enforce it by code? (Using max(10, xxx))

Reply:
That's a value from the LevelDB code, see org.iq80.leveldb.impl.DbImpl#DbImpl
```
        // Reserve ten files or so for other uses and give the rest to TableCache.
        int tableCacheSize = options.maxOpenFiles() - 10;
        tableCache = new TableCache(databaseDir, tableCacheSize, new InternalUserComparator(internalKeyComparator), options.verifyChecksums());
```
I don't really want to bring it to the CDAP code as it's pretty implementation-specific and we may change LevelDB implementation to something else, but I still want to warn about it.
Not sure though. A possible option to add this 10 to the user value, but we would need to explain it in the code

Reply:
Done